app/javascript/components/Results/BestBets/GraphicsBestBet.tsx

```diff
@@ -49,14 +55,27 @@ export const GraphicsBestBet = ({ title, titleUrl, imageUrl, imageAltText, links
         <Grid col={true} className='result-meta-data'>
           <div className='graphics-best-bets-title result-title'>
             {titleUrl ? (
-              <h2 className='result-title-label'><a href={titleUrl} className='result-title-link'>{parse(title)}</a></h2>) : <h2 className='result-title-label'>{parse(title)}</h2>}
+              <h2 className='result-title-label'>
+                <ResultTitle 
```

Comment:
This can be like
```
<ResultTitle 
 url={titleUrl}
 className='result-title-link'
clickTracking={() => clickTracking(affiliate, moduleCode.bestBetsGraphics, query, 1, titleUrl, vertical)}
>
```
Refer implementation at https://github.com/GSA/search-gov/pull/1493/files